.github/workflows/codeql-analysis.yml

```diff
@@ -12,10 +12,8 @@
 name: "CodeQL"
 
 on:
-  push:
-    branches: [main]
   pull_request:
-    # The branches below must be a subset of the branches above
+  push:
     branches: [main]
   schedule:
     - cron: '37 21 * * 4'
```

Comment:
Maybe we can get rid of this setting since this action run everytime we create a PR on main